back-end/api/quickstart/views.py

```diff
@@ -162,13 +162,19 @@ class CommentViewSet(viewsets.ModelViewSet):
     lookup_field = 'id'
 
     def retrieve(self, request, author, post):
+        queryset = Comment.objects.filter(post=post)
```

Comment:
Shouldn't this post be the actual post object not the id? since its a foreign key

Reply:
This seems to work (Django is smart enough to take in the ID).